src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/README.md

```diff
@@ -0,0 +1,43 @@
+## Unofficial MSBuild Project SDK support
+
+This is an unofficial package which introduces a Xamarin.Android [MSBuild Project SDK](https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk).
+In some ways this is a standalone implementation / extension of MSBuild.Sdk.Extras for Xamarin.Android.
+
+Default Android related file globbing behavior is defined in `Microsoft.Android.Sdk.DefaultItems.props`.
```

Comment:
Should this entire file go in this `DotNet5.md`?

https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/DotNet5.md

Then maybe we don't even need a `README.md` here?